Assets/Scripts/Door.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class Door : MonoBehaviour
+{
+    [SerializeField]
+    private string scenetoLoad;
+
+    private bool isPlayerInTrigger;
+
+    // cant use this bc it triggers the door twice, bc the player has two colliders
+    //private void OnTriggerStay2D(Collider2D collision)
+    //{
+    //    if (collision.CompareTag("Player"))
+    //    {
+    //        if (Input.GetButtonDown("Activate"))
+    //        {
+    //            Debug.Log("Player activated door!");
+    //        }
+           
+    //    }
+    //}
+
+    private void OnTriggerEnter2D(Collider2D collision)
+    {
+        if (collision.CompareTag("Player"))
+        {
+            isPlayerInTrigger = true;
+        }
+    }
+
+    private void nTriggerExit2D(Collider2D collision)
+    {
+        isPlayerInTrigger = false;
+    }
+
+    private void Update()
+    {
+        if (Input.GetButtonDown("Activate") && isPlayerInTrigger)
+        {
+            Debug.Log("Player activated door!");
+            SceneManager.LoadScene(scenetoLoad);
+        }
+    }
+}
```

Comment:
Hmm, not sure what this error is. Seems like a missing New Line? Might just be a github error.